addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -288,7 +288,7 @@ registerModel({
                 try {
                     const audioStream = await browser.navigator.mediaDevices.getUserMedia({ audio: this.messaging.userSetting.getAudioConstraints() });
                     audioTrack = audioStream.getAudioTracks()[0];
-                } catch (e) {
+                } catch (_e) {
```

Comment:
This looks like noise, make them empty `catch` instead?

Reply:
That's what I ask about above, but the JS team is understandably somewhat wary of making such a change as it's not that old a feature (it's ES2019).